packages/side-nav/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import SideNav from './SideNav';
+import SideNavItem, { SideNavItemProps } from './SideNavItem';
+import SideNavGroup from './SideNavGroup';
+
+export { SideNav, SideNavItem, SideNavGroup, SideNavItemProps };
```

Comment:
I'm wondering if rather than add this to our public export API we just let the test import the props directly from the module.

Reply:
Clever. I didn't think of using two import statements like that.